app/assets/stylesheets/govuk-component/_breadcrumbs.scss

```diff
@@ -15,4 +15,32 @@
     color: $secondary-text-colour;
     text-decoration: none;
   }
+
+  @include media-down(mobile) {
+    &.collapse-on-mobile li {
+      display: none;
+
+      &.parent-taxon {
+        background-image: none;
+        display: block;
+        margin-left: 0;
+        padding-left: 14px;
+        position: relative;
+
+        &:before {
+          border-bottom: 5px solid transparent;
```

Comment:
Hm. In that case I'd rather we import the [actual Sass](https://github.com/alphagov/govuk_elements/blob/c9af6fb107c86955fcb105d35ce3c5647fa870ab/public/sass/elements/_elements-typography.scss#L184-L231) rather than copy-paste.